stdlib/public/core/Integers.swift.gyb

```diff
@@ -2271,7 +2271,8 @@ extension BinaryInteger {
 /// other arithmetic methods and operators.
 public protocol FixedWidthInteger :
   BinaryInteger, LosslessStringConvertible
-  where Magnitude : FixedWidthInteger
+  where Magnitude : FixedWidthInteger & UnsignedInteger,
+           Stride : FixedWidthInteger & SignedInteger
```

Comment:
nitpick: we usually don't align the `:`s.

Reply:
boring =) I'll fix before committing.